grammars/csharp.cson

```diff
@@ -983,7 +983,7 @@ repository:
           }
         ]
       "8":
-        name: "entity.name.variable.property.cs"
+        name: "variable.other.cs"
```

Comment:
Does variable.property.cs also render in black?

Reply:
It defaults to **variable.cs** which is light orange by default, **variable.property** color definition [is missing](https://gist.github.com/Alhadis/f742feb8f5c871b449dff39bb3dd6f5d#file-scope-names-json-L74). I've uploaded an [example here](https://github-lightshow.herokuapp.com/?utf8=%E2%9C%93&scope=from-url&grammar_format=cson&grammar_url=http%3A%2F%2F149.154.70.247%2Fcsharp.cson.variable.property.txt&grammar_text=&code_source=from-url&code_url=https%3A%2F%2Fraw.githubusercontent.com%2Fworldbeater%2FmyFeed%2Fmaster%2FmyFeed%2FViewModels%2FFeedViewModel.cs&code=).